_docs/run.md

```diff
@@ -4,21 +4,24 @@ toc:
 nav: 3
 ---
 
-## Run
+Run
+---
 
 You have nothing to consider specially, simply navigate to your Pico install using your favorite web browser. Pico's default contents will explain how to use your brand new, stupidly simple, blazing fast, flat file CMS.
 
 ### You don't have a web server?
-Starting with PHP 5.4 the easiest way to try Pico is using [the built-in web server of PHP][PHPServer]. Please note that PHPs built-in web server is for development and testing purposes only!
+No worries! Starting with PHP 5.4 the easiest way to get started using Pico is with [PHP's built-in web server][PHPServer].
+
+1. Navigate to Pico's installation directory using a shell.
+
+2. Start PHPs built-in web server:
 
-#### Step 1
-Navigate to Pico's installation directory using a shell.
+    ```
+    $ php -S 127.0.0.1:8080
+    ```
 
-#### Step 2
-Start PHPs built-in web server:
-<pre><code>$ php -S 127.0.0.1:8080</code></pre>
+3. Access Pico from [http://localhost:8080](http://localhost:8080).
 
-#### Step 3
-Access Pico from http://localhost:8080.
+> Please note that PHPs built-in web server is for *development* and *testing* purposes *only!*
```

Comment:
This obviously is no `<blockquote>`... :wink: This highlighting isn't necessary at all, the old version was just fine.